sdk/python/jobs/pipelines/1l_flow_in_pipeline/additional_includes/convert_to_dict.py

```diff
@@ -0,0 +1,12 @@
+import json
```

Comment:
let's remove this additional includes in sdk example  and make it depends on the cli example files

Reply:
I haven't done this as the relative path will be extremely long. hmm, will try to optimize this